src/app/app.component.ts

```diff
@@ -60,7 +60,7 @@ export class AppComponent implements OnInit {
     this.checkLoginStatus();
     this.listenForLoginEvents();
 
-    this.swUpdate.available.subscribe(async res => {
+    this.swUpdate.versionUpdates.subscribe(async res => {
```

Comment:
`available` became [deprecated](https://stackoverflow.com/questions/77848425/angular-17-swupdate-property-available-does-not-exist-on-type-swupdate) and replaced with `versionUpdates`.